CppCoreGuidelines.md

```diff
@@ -20308,6 +20308,14 @@ For writing to a file, there is rarely a need to `flush`.
 
 ##### Note
 
+For string streams (specifically `ostringstream`), the insertion of an `endl` is entirely equivalent
+to the insertion of a `'\n'` character, but also in this case, `endl` might be significantly slower.
+
+`endl` does *not* take care of producing a platform specific end-of-line sequence (like "\r\n" on
+Windows). So for a string stream, `s << endl` just inserts a *single* character, `'\n'`.
```

Comment:
or `\n\r` for Mac ;)

Reply:
> or `\n\r` for Mac ;)

@BenjamenMeyer You mean '\r', right? But hasn't Mac moved towards using just a single `\n`, nowadays? Otherwise, are there still actively used C++11 Standard Library implementations for Mac whose file output streams produce a '\r' on each end-of-line?

I did notice the smiley in your comment, but I just want to be entirely sure that it's OK _not_ to mention the old Mac '\r' in the proposed text.

Reply:
@N-Dekker AFAIK mac is still doing the `\n\r` - now, perhaps it changes from application to application, but I've still seen it in recent years. The new text called out the Windows behavior as an example; it could be worth noting (legacy or otherwise) the Mac behavior too.

Reply:
@jwakely @BenjamenMeyer Thanks, I won't mention old Macs line endings, if it isn't really necessary. So I also think the PR is OK now.